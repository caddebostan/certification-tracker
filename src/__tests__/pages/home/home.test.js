import React from 'react';
import { shallow } from 'enzyme';

import Dashboard from '../../../views/dashboard/dashboard';

it('renders correctly', () => {
  const wrapper = shallow(<Dashboard />);

  expect(wrapper).toMatchSnapshot();
});
