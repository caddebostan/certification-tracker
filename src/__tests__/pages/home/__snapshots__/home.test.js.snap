// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Dashboard
  classes={
    Object {
      "successText": "Dashboard-successText-1",
      "upArrowCardCategory": "Dashboard-upArrowCardCategory-2",
    }
  }
/>
`;
